shared/reducers/index.js

```diff
@@ -73,9 +73,9 @@ if (__DEV__) {
 }
 
 export default function (state: Object = {}, action: any): State {
-  // Reset our state if we logout
+  // Reset most of out state if we logout
   if (action.type === logoutDone) {
-    state = {}
```

Comment:
Does this actually work? I thought I had a pr (maybe it never went in) where I listen for logout in the root reducer and blow it away there